src/DatePicker/createCalendar.js

```diff
@@ -1,7 +1,8 @@
 import flatpickr from "flatpickr";
-import l10n from "flatpickr/dist/l10n/index.js";
 import rangePlugin from "flatpickr/dist/plugins/rangePlugin";
 
+var l10n
```

Comment:
Maybe we could declare it following ES2015:

```js
let l10n
```

Reply:
Correct, sorry about that. It is done.